src/Moryx.Asp.Extensions/Exception/MoryxExceptionFilter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System.Linq;
+
+namespace Moryx.Asp.Extensions.Exception
+{
+    public class MoryxExceptionFilter : IExceptionFilter
+    {
+        public void OnException(ExceptionContext context)
+        {
+            context.HttpContext.Response.StatusCode = StatusCodes.Status500InternalServerError;
+            string headers = string.Join(" \r\n ", context.HttpContext.Request.Headers.Select(h => $"{h.Key}: {h.Value}"));
```

Comment:
For 500, in postman it showed the request headers, so I added them to the exception